aliases/available/general.aliases.bash

```diff
@@ -11,7 +11,7 @@ alias l1='ls -1'
 
 alias _="sudo"
 
-if [ $(uname) = "Linux" ]
+if [ $(uname) = "Linux" ] || brew ls --versions coreutils > /dev/null
```

Comment:
I agree. I thought of this after I submitted this PR. I looked for a way to close this but then figured I would wait and see what someone said, maybe get a better way to do it.

Thank you!